lib/redis-result-transformation-functions.js

```diff
@@ -0,0 +1,17 @@
+// Reading from Redis always returns a flat array
+// this file contains rules for transforming these
+// arrays into arrays of Juttle points given
+// a command and its output
+
+module.exports = {
+    hmget: function(command, result) {
+        var output = {};
+        var fields = command.slice(2);
+        var values = result[0];
+        for (var i = 0; i < fields.length; i++) {
+            output[fields[i]] = values[i];
+        }
```

Comment:
Out of curiosity does something like `node-redis` handle these transformations for you?